doc/config/common_io

```diff
@@ -0,0 +1,24 @@
+# Include common configuration options.
+@INCLUDE_PATH = doc/config
+@INCLUDE = common
+
+# Basic project information.
+PROJECT_NAME = "Common I/O"
+PROJECT_BRIEF = "AWS IoT Common I/O library"
+
+# Library documentation output directory.
+HTML_OUTPUT = common_io
+
+# Generate Doxygen tag file for this library.
+GENERATE_TAGFILE = doc/output/common_io.tag
+
+# Directories containing library source code.
+INPUT = doc/lib/ \
+        libraries/abstractions/common_io/include
+
+
+# Library file names.
+FILE_PATTERNS = iot_i2c.h iot_uart.h
```

Comment:
Are we only including these two in the docs?

Reply:
No. There will be more. This is just initial commit.